pkg/client/rootd/session.go

```diff
@@ -626,6 +631,7 @@ func (s *Session) checkConnectivity(ctx context.Context, info *manager.ClusterIn
 		return
 	}
 	conn.Close()
+	// NOTE: we can access pod, but can we access service?
```

Comment:
Please remove this note. It isn't helpful. A `TODO: <a more elaborated action>` would be different.